doc/api/vm.md

```diff
@@ -304,29 +304,6 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
-## vm.runInDebugContext(code)
-<!-- YAML
-added: v0.11.14
--->
```

Comment:
@joshgav I don’t think you need to do that, the YAML changes list is for behavioural changes, not doc changes, and that it’s deprecated should be obvious from the stability marker :)